src/app/chip_data_model.gni

```
@@ -248,278 +248,31 @@ template("chip_data_model") {
       defines = []
     }
 
+    # Clusters are expected to have the following format:
+    #   - A `BUILD.gn` file that defines generic targets. A dependency
```

"generic" is pretty unclear here. This defines targets that do not depend on the generated application configuration, right?
```suggestion
    #   - A `BUILD.gn` file that defines targets that do not depend on the specific application configuration. A dependency
```